dash/extract-meta.js

```diff
@@ -702,6 +708,23 @@ function gatherComponents(sources, components = {}) {
                 }
             }
 
+            const isArrowFunction = typeSymbol.declarations && typeSymbol.declarations[0].kind === ts.SyntaxKind.ArrowFunction
+            const isClass = ts.isClassDeclaration(declaration);
+
+            if (!isArrowFunction && !isClass) {
```

Comment:
I think it could be a regular `function(props: Props) {return <div></div>;` not only an arrow function.

Reply:
Good spot I'll make the changes tonight